commands/motor/MotorControlAccelCap.java

```diff
@@ -1,15 +1,14 @@
 package org.usfirst.frc4904.standard.commands.motor;
 
-
 import org.usfirst.frc4904.standard.LogKitten;
 import org.usfirst.frc4904.standard.custom.controllers.Controller;
 import org.usfirst.frc4904.standard.subsystems.motor.Motor;
 import org.usfirst.frc4904.standard.subsystems.motor.PositionSensorMotor;
 import edu.wpi.first.wpilibj.command.Command;
 
 /**
- * Controls a Motor directly from a Controller (e.g. Joystick or Xbox)
- * Has an acceleration cap (but not deceleration cap)
+ * Controls a Motor directly from a Controller (e.g. Joystick or Xbox) Has an
+ * acceleration cap (but not deceleration cap)
  *
  */
 public class MotorControlAccelCap extends Command {
```

Comment:
This class needs updating to the new wpilibj2 `Command`

Reply:
Not a file we should have edited so no need to fix.